### PR TITLE
fix(rhosak): change typo in payload key

### PR DESCRIPTION
**Pull request by machi1990**
/cc @gwenneg

### CODE REVIEW

File: backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/RhosakEmailAggregator.java [hunk 1]
```diff
@@ -15,7 +15,7 @@
     private static final String SERVICE_DISRUPTION_EVENT_TYPE = "disruption";
     private static final String CONTEXT = "context";
     private static final String EVENTS = "events";
-    private static final String PAYLOAD = "events";
+    private static final String PAYLOAD = "payload";
```
Comment (machi1990) [inline backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/RhosakEmailAggregator.java:18]: good catch @gwenneg 

I'll be happy if you can give this file another round of review. 

Normally, I should have put some unit tests into it but I am still trying to find my way into the aggregator part.

Comment (gwenneg) [reply backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/RhosakEmailAggregator.java:18]: I had a look at the file. I'm not (yet 😄) an aggregator expert but it looks good to me.

**Review by gwenneg — APPROVED** [review]
LGTM

File: backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/RhosakEmailAggregator.java (hunk 1 above)
Comment (machi1990) [reply backend/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/RhosakEmailAggregator.java:18]: Thank you a lot @gwenneg

**Pull request merged**